.github/workflows/ci.yaml

```diff
@@ -146,6 +146,17 @@ jobs:
             -w compute_xsmall \
             -d METADATA
 
+      - name: future grants 
+        shell: bash
+        run: |
+          schemachange \
+            -f admin/future_grants \
+            -a $SNOWFLAKE_ACCOUNT \
+            -u $SNOWFLAKE_USER \
+            -r SECURITYADMIN \
```

Comment:
I just logged onto the DPE_SERVICE account and noticed it has SYSADMIN access but no SECURITYADMIN access, so I feel compelled to ask: Are we sure the `$SNOWFLAKE_ACCOUNT` we use for these schemachange operations has access to SECURITYADMIN role? I don't see this role used elsewhere so asking just in case.

Reply:
Good catch. Looks like we've never had to use SECURITYADMIN to manage grants before. Everything has been done as the owner of an object granting privileges upon that object. I'll open another PR.